SSPSolution/SSPSolution.sln

```diff
@@ -32,41 +32,67 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "physicsDLL", "physicsDLL\ph
+		{9BFF2156-5379-46CE-9BC9-35BD895C4ECD}.Debug|Any CPU.ActiveCfg = Debug|Win32
 		{9BFF2156-5379-46CE-9BC9-35BD895C4ECD}.Debug|x86.ActiveCfg = Debug|Win32
 		{9BFF2156-5379-46CE-9BC9-35BD895C4ECD}.Debug|x86.Build.0 = Debug|Win32
+		{9BFF2156-5379-46CE-9BC9-35BD895C4ECD}.Release|Any CPU.ActiveCfg = Release|Win32
 		{9BFF2156-5379-46CE-9BC9-35BD895C4ECD}.Release|x86.ActiveCfg = Release|Win32
 		{9BFF2156-5379-46CE-9BC9-35BD895C4ECD}.Release|x86.Build.0 = Release|Win32
+		{59269BE3-152C-4D71-BF1A-B4B0312ADBB2}.Debug|Any CPU.ActiveCfg = Debug|Win32
 		{59269BE3-152C-4D71-BF1A-B4B0312ADBB2}.Debug|x86.ActiveCfg = Debug|Win32
 		{59269BE3-152C-4D71-BF1A-B4B0312ADBB2}.Debug|x86.Build.0 = Debug|Win32
 		{59269BE3-152C-4D71-BF1A-B4B0312ADBB2}.Debug|x86.Deploy.0 = Debug|Win32
+		{59269BE3-152C-4D71-BF1A-B4B0312ADBB2}.Release|Any CPU.ActiveCfg = Release|Win32
 		{59269BE3-152C-4D71-BF1A-B4B0312ADBB2}.Release|x86.ActiveCfg = Release|Win32
 		{59269BE3-152C-4D71-BF1A-B4B0312ADBB2}.Release|x86.Build.0 = Release|Win32
+		{FA1FEC57-5873-49E7-8F1F-AB3941D1F52A}.Debug|Any CPU.ActiveCfg = Debug|Win32
 		{FA1FEC57-5873-49E7-8F1F-AB3941D1F52A}.Debug|x86.ActiveCfg = Debug|Win32
 		{FA1FEC57-5873-49E7-8F1F-AB3941D1F52A}.Debug|x86.Build.0 = Debug|Win32
+		{FA1FEC57-5873-49E7-8F1F-AB3941D1F52A}.Release|Any CPU.ActiveCfg = Release|Win32
 		{FA1FEC57-5873-49E7-8F1F-AB3941D1F52A}.Release|x86.ActiveCfg = Release|Win32
 		{FA1FEC57-5873-49E7-8F1F-AB3941D1F52A}.Release|x86.Build.0 = Release|Win32
+		{FCB8D293-A5CA-4036-85AE-5CFE3053F7CA}.Debug|Any CPU.ActiveCfg = Debug|Win32
 		{FCB8D293-A5CA-4036-85AE-5CFE3053F7CA}.Debug|x86.ActiveCfg = Debug|Win32
 		{FCB8D293-A5CA-4036-85AE-5CFE3053F7CA}.Debug|x86.Build.0 = Debug|Win32
+		{FCB8D293-A5CA-4036-85AE-5CFE3053F7CA}.Release|Any CPU.ActiveCfg = Release|Win32
 		{FCB8D293-A5CA-4036-85AE-5CFE3053F7CA}.Release|x86.ActiveCfg = Release|Win32
 		{FCB8D293-A5CA-4036-85AE-5CFE3053F7CA}.Release|x86.Build.0 = Release|Win32
+		{B12702AD-ABFB-343A-A199-8E24837244A3}.Debug|Any CPU.ActiveCfg = Debug|Win32
 		{B12702AD-ABFB-343A-A199-8E24837244A3}.Debug|x86.ActiveCfg = Debug|Win32
+		{B12702AD-ABFB-343A-A199-8E24837244A3}.Release|Any CPU.ActiveCfg = Release|Win32
 		{B12702AD-ABFB-343A-A199-8E24837244A3}.Release|x86.ActiveCfg = Release|Win32
 		{B12702AD-ABFB-343A-A199-8E24837244A3}.Release|x86.Build.0 = Release|Win32
+		{0E7D331B-B435-4A7E-8074-09225FEC9CDB}.Debug|Any CPU.ActiveCfg = Debug|Win32
 		{0E7D331B-B435-4A7E-8074-09225FEC9CDB}.Debug|x86.ActiveCfg = Debug|Win32
 		{0E7D331B-B435-4A7E-8074-09225FEC9CDB}.Debug|x86.Build.0 = Debug|Win32
 		{0E7D331B-B435-4A7E-8074-09225FEC9CDB}.Debug|x86.Deploy.0 = Debug|Win32
+		{0E7D331B-B435-4A7E-8074-09225FEC9CDB}.Release|Any CPU.ActiveCfg = Release|Win32
 		{0E7D331B-B435-4A7E-8074-09225FEC9CDB}.Release|x86.ActiveCfg = Release|Win32
 		{0E7D331B-B435-4A7E-8074-09225FEC9CDB}.Release|x86.Build.0 = Release|Win32
+		{C07D2EAA-EBAE-4781-A16F-50B4A1822AF0}.Debug|Any CPU.ActiveCfg = Debug|Win32
```

Comment:
I could look into these tomorrow morning. Looks like you've accidently added another build definition?